theme-dev/MainStrings.md

```diff
@@ -0,0 +1,64 @@
+<h1 align="center">Main Strings</h1>
+
+**DO NOT CHANGE THE JSON FILE DIRECTLY BECAUSE MOST OF THE TIME YOU FUCK SOMETHING UP**
```

Comment:
this should be in the main readme, not here also please add small introduction to themes, how to use them and how to make them

Reply:
I dont exactly understand the part on moving the strings into the main readme, do I make it like I did on the old legacy docu?

Reply:
I am kind of stupid rn to make an introduction to themes since I cant think of a good way to start in a "professional" way

Reply:
> I dont exactly understand the part on moving the strings into the main readme, do I make it like I did on the old legacy docu?

I mean the "Do not change the json file directly" stuff and so on, this should be in the main readme not here

Reply:
So basically warnings and so on, got it!